### DISCUSSION TITLE
我用的中转api，无法使用提取文章关键词汇的功能

### DISCUSSION THREAD

**Discussion by girlfriendandme** [ZuodaoTech/everyone-can-use-english#586] [Q&A]
望增加对应的功能，可以在提取关键词汇上选择模型，使用中转api。

**Comment by an-lee** [comment]
应该是可以的，再确认下你的中转 api 正常可用

**Comment by girlfriendandme** [comment]
我也不知道为什么，智能助手里能用中转api，但是在文章提取词汇的时候，就显示提取失败。

![image](https://github.com/xiaolai/everyone-can-use-english/assets/48598963/b805d203-a859-4b21-ae02-851ec85d940f)

![image](https://github.com/xiaolai/everyone-can-use-english/assets/48598963/f4db6981-96bf-488b-ad2c-a14284dc541d)

**Comment by girlfriendandme** [comment]
> 应该是可以的，再确认下你的中转 api 正常可用


我也不知道为什么，智能助手里能用中转api，但是在文章提取词汇的时候，就显示提取失败。
